lib/components/map-a-domain-component.js

```diff
@@ -0,0 +1,15 @@
+import { By } from 'selenium-webdriver';
+
+import BaseContainer from '../base-container.js';
+import * as driverHelper from '../driver-helper';
+
+export default class MapADomainComponent extends BaseContainer {
+	constructor( driver ) {
+		super( driver, By.css( '.transfer-domain-step' ) );
+	}
+
+	selectManuallyConnectExistingDomain() {
+		const manuallyConnectSelector = By.css( '#primary > main > span > div > div:nth-child(2) > div > div > p > a:nth-child(1)' );
```

Comment:
We try to avoid CSS selectors like this because of the nested chain they're fragile and easy to break, since if someone changes one element in the chain it will no longer work.

I think something like `.card.transfer-domain-step__map-option a[href="#"]` will work, which finds the `a` (link) with a link of # which is a child of the card. I use Chrome to inspect elements and use command-f in the elements window where you can type a CSS selector to make sure it works (and is unique):

<img width="1090" alt="screen shot 2018-01-25 at 4 19 04 pm" src="https://user-images.githubusercontent.com/128826/35373717-80330a8e-01eb-11e8-9f29-5c55f79df0b9.png">

Reply:
Thanks for the explanation and the tip here! Also thanks for finding the right selector. Fixed.